pkg/log/status.go

```diff
@@ -294,6 +294,16 @@ func Title(firstLine, secondLine, thirdLine string) {
  \__/%s`, blue(firstLine), secondLine, thirdLine, "\n")
 }
 
+// Stitle is the same as Title but returns the string instead
+func Stitle(firstLine, secondLine, thirdLine string) string {
```

Comment:
If `Stitle` is the same as `Title` but just returning the string, why not make `Title` use the result of `Stitle` then? This would help reduce code duplication in this particular case, I think.

```go
func Title(firstLine, secondLine, thirdLine string) {
	fmt.Fprintf(GetStdout(), Stitle(firstLine, secondLine, thirdLine))
}
```

Reply:
You're right! Thank you.

Reply:
So unfortunately if I change it to the above suggestion, it results in a lint error:

```sh
pkg/log/status.go:289:2: SA1006: printf-style function with dynamic format string and no further arguments should use print-style function instead (staticcheck)
```

We'll have to keep it to how it was before.

Reply:
Ah, I see. As suggested in the [`Staticcheck` rule](https://staticcheck.io/docs/checks#SA1006), how about fixing this with `Fprint` instead, since `Stitle` returns the string and not a format string?
```go
func Title(firstLine, secondLine, thirdLine string) {
	fmt.Fprint(GetStdout(), Stitle(firstLine, secondLine, thirdLine))
}
```